frontend/src/metabase/query_builder/components/RunButton.tsx

```diff
@@ -11,10 +11,11 @@ interface RunButtonProps {
   className?: string;
   isRunning: boolean;
   isDirty: boolean;
-  compact?: boolean;
```

Comment:
1. `compact` is always `true
2. `borderless` is not passed
3. `medium`, `onlyIcon`, `iconSize` are actually passed but not mentioned in the types